manifests/windows/agent6.pp

```diff
@@ -16,8 +16,13 @@
   Enum['present', 'absent'] $ensure = 'present',
 ) inherits datadog_agent::params {
 
-  $msi_full_path = "${msi_location}/datadog-agent-6-${agent_version}.amd64.msi"
-  $msi_source = "${baseurl}ddagent-cli-${agent_version}.msi"
+  if $agent_version == "latest" {
+    $msi_source = "${baseurl}datadog-agent-6-latest.amd64.msi"
```

Comment:
wondering if we should maybe use `join` to build the URIs?

Reply:
I haven't found a version of `join` specific for URLs, so I don't think it's going to be better than this actually.